MODULE.bazel

```diff
@@ -1,2 +1,51 @@
-# TODO: migrate all dependencies from WORKSPACE to MODULE.bazel
-# https://github.com/protocolbuffers/protobuf/issues/14313
+"""Bazel dependencies
+
+Currently only used by external users via the Bazel Central Registry.
+TODO(alexeagle): empty the WORKSPACE file and enable bzlmod for this repo.
+
+Note, originally this file existed as a patch:
+https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/protobuf/23.1/patches/0001-Add-MODULE.bazel.patch
+"""
+module(
+    name = "protobuf",
+    compatibility_level = 1,
+    version = "23.1",
```

Comment:
We're currently working on 26.0 (you could say main is on 27.0), it seems like all of these dependencies are stale